src/auth/oauth2client.ts

```diff
@@ -892,32 +925,44 @@ export class OAuth2Client extends AuthClient {
   /**
    * Gets federated sign-on certificates to use for verifying identity tokens.
    * Returns certs as array structure, where keys are key ids, and values
-   * are PEM encoded certificates.
+   * are certificates in either PEM or JWK format.
    * @param callback Callback supplying the certificates
    */
   getFederatedSignonCerts(): Promise<FederatedSignonCertsResponse>;
-  getFederatedSignonCerts(callback: GetFederatedSignonCertsCallback): void;
-  getFederatedSignonCerts(callback?: GetFederatedSignonCertsCallback):
+  getFederatedSignonCerts(
+      format: CertificateFormat,
+      callback: GetFederatedSignonCertsCallback): void;
+  getFederatedSignonCerts(
+      format?: CertificateFormat, callback?: GetFederatedSignonCertsCallback):
       Promise<FederatedSignonCertsResponse>|void {
+    if (!format) {
```

Comment:
You can just set this in the function signature instead of making it optional and doing this check below:

`getFederatedSignonCerts(format: CertificateFormat = CertificateFormat.PEM ...`

Reply:
still learning typescript :)